modules/network-observability-configuring-netobserv-dashboard.adoc

```diff
@@ -0,0 +1,26 @@
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-netobserv-dashboard_{context}"]
+= Configuring viewable metrics for the Netobserv dashboard
+You can add or remove metrics from the *Netobserv* dashboard by editing the `ignoreTags` list in the `FlowCollector` `spec.processor.metrics` parameter. 
+
+.Procedure
+. In the web console, navigate to Operators → Installed Operators.
+. Under the Provided APIs heading for the NetObserv Operator, select Flow Collector.
+. Select cluster then select the YAML tab.
+. Edit the `ignoreTags` list to add or remove metrics from the dashboard, as in the following YAML sample:
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  processor:
+    metrics:
+      ignoreTags: [egress, ingress, flows, bytes, packets, namespaces, nodes, workloads]
+----
+
+For more information about available tags, see the _Flow Collector API Reference_
```

Comment:
I think we can be more accurate about how tags work here. Copying the metrics list that you wrote in another file, we can tell which tag they have:

 * *Top flow rates per source and destination namespaces*: tagged "flows" and "namespaces"
 * *Top byte rates emitted per source and destination nodes*: tagged "egress", "bytes" and "nodes"
 * *Top byte rates received per source and destination nodes*: tagged "ingress", "bytes" and "nodes"
 * *Top byte rates emitted per source and destination workloads*: tagged "egress", "bytes" and "workloads"
 * *Top byte rates received per source and destination workloads*: tagged "ingress", "bytes" and "workloads"
 * *Top packet rates emitted per source and destination workloads*: tagged "egress", "packets" and "workloads"
 * *Top packet rates received per source and destination workloads*: tagged "ingress", "packets" and "workloads"

Ignoring a label will ignore all metrics having that label. For instance, when ignoring "packets", "nodes" and "egress", only remain _Top flow rates per source and destination namespaces_ and _Top byte rates received per source and destination workloads_

Reply:
Okay good feedback. I'll rework this a bit to better describe how the tags work and relate it to the tag descriptions in the Overview.